pypi_formula_mappings.json

```
@@ -230,6 +230,9 @@
   "nox": {
     "exclude_packages": ["six"]
   },
+  "nvchecker": {
+    "extra_packages": ["packaging"]
+  },
```

Probably better to enable feature directly.
```suggestion
  "nvchecker": "nvchecker[pypi]",
```

---

Thanks, it looks better. Why put `nvchecker[pypi]` here? Is there documentation?